src/base/events.js

```diff
@@ -282,7 +282,15 @@ Events.PLAYER_SEEK = 'seek'
  * @event PLAYER_ERROR
  * @param {Object} error the error
  */
-Events.PLAYER_ERROR = 'error'
+Events.PLAYER_ERROR = 'playererror'
+/**
+ * Fired when there is an error
+ *
+ * @event ERROR
+ * @param {Object} error
```

Comment:
Can you describe/document the proposed error object here? Like we do at [Player's constructor](https://github.com/jhonatangomes/clappr/blob/c5e5fcf174c9d60135750608d4e3af5c4f4aa45e/src/components/player.js#L152-L160)?

Reply:
yes, done.